css/styles.css

```diff
@@ -0,0 +1,67 @@
+.cat {
+    width: 25%;
+    margin: 10px 0 10px 0;
+}
+
+.featured-work {
+    color: #ACACAC;
```

Comment:
#### Critical :x:
Remember that the correct form to use hexadecimal colors is like this: "#acacac", You can see more in the css style guide in this link http://udacity.github.io/frontend-nanodegree-styleguide/css.html.